critiquebrainz/frontend/views/review.py

```diff
@@ -67,6 +67,16 @@ def browse():
 
 # TODO(psolanki): Refactor this function to remove PyLint warning.
 # pylint: disable=too-many-branches
```

Comment:
is this pylint-disable from the `def entity` function? If injecting this function into the middle of the declaration/function doesn't change the results of pylint then we can remove the comment